run_operator_locally.sh

```diff
@@ -165,9 +165,14 @@ function forward_ports(){
     local operator_pod
     operator_pod=$(kubectl get pod -l name=postgres-operator -o jsonpath={.items..metadata.name})
 
-    # runs in the background to keep current terminal responsive
-    # stdout redirect removes the info message about forwarded ports; the message sometimes garbles the cli prompt
-    kubectl port-forward "$operator_pod" "$LOCAL_PORT":"$OPERATOR_PORT" &> /dev/null &
+    # Run in the background to keep current terminal responsive. Hide stdout
+    # because otherwise there is a note about each TCP connection. Do not hide
+    # stderr so port-forward setup errors can be debugged. Sometimes the
+    # port-forward setup fails because expected state isn't achieved yet. in
+    # that, case a bit of retrying helps.
+
+    retry "kubectl port-forward "$operator_pod" "$LOCAL_PORT":"$OPERATOR_PORT" > /dev/null &" \
+        "attempt to create port-forward"
```

Comment:
Note: that retry loop of course does not work because the process is sent to the background. Will work on a refined patch.